docs/chat_message_history.ipynb

```diff
@@ -4,55 +4,129 @@
+            "metadata": {},
+            "source": [
+                "## Basic Usage"
             ]
         },
         {
             "cell_type": "code",
-            "execution_count": 3,
-            "metadata": {
-                "tags": []
-            },
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "from langchain_google_bigtable import (\n",
+                "    BigtableChatMessageHistory,\n",
+                ")\n",
+                "\n",
+                "message_history = BigtableChatMessageHistory(\n",
+                "    instance_id=\"my-instance\",\n",
+                "    table_id=\"my-table\",\n",
+                "    session_id=\"user-session-id\",\n",
+                ")\n",
+                "\n",
+                "message_history.add_user_message(\"hi!\")\n",
+                "message_history.add_ai_message(\"whats up?\")"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [
+                {
+                    "data": {
+                        "text/plain": [
+                            "[HumanMessage(content='hi!'),\n",
+                            " HumanMessage(content='hi!'),\n",
+                            " AIMessage(content='whats up?')]"
+                        ]
+                    },
+                    "metadata": {},
+                    "output_type": "display_data"
+                }
+            ],
+            "source": [
+                "message_history.messages"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "## Custom client\n",
+                "The client created by default is the default client, using only admin=True option. To use a non-default, a [custom client](https://cloud.google.com/python/docs/reference/bigtable/latest/client#class-googlecloudbigtableclientclientprojectnone-credentialsnone-readonlyfalse-adminfalse-clientinfonone-clientoptionsnone-adminclientoptionsnone-channelnone) can be passed to the constructor."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
             "outputs": [],
             "source": [
-                "from PACKAGE import LOADER"
+                "from google.cloud import bigtable\n",
+                "\n",
+                "custom_client_message_history = BigtableChatMessageHistory(\n",
+                "    instance_id=\"my-instance\",\n",
+                "    table_id=\"my-table\",\n",
+                "    client=bigtable.Client(...),\n",
+                ")"
             ]
         },
         {
             "cell_type": "markdown",
             "metadata": {},
             "source": [
-                "## Basic Usage"
+                "## Cleaning up\n",
```

Comment:
Would it flow better to add this section to the bottom of "Basic Usage"? It would then be `initialize ChatMessageHistory -> add messages -> show messages -> clear messages` This way you don't need to initialize another class object and can just use the one you already have...

```python
message_history.clear()
```